### PR TITLE
Mailing group, tag

### PR DESCRIPTION
**Pull request by semseysandor**
In this PR the extension installs a mailing group and tag which can be used for sending monitoring emails and labeling contacts.

Next step is an API to add contacts to this group.

### CODE REVIEW

File: tests/phpunit/CRM/DeliveryMonitor/UpgraderTest.php [hunk 1]
```diff
@@ -0,0 +1,128 @@
+<?php
+
+use Civi\Api4\Group;
+use Civi\Api4\Tag;
+use Civi\Test;
+use Civi\Test\HeadlessInterface;
+use PHPUnit\Framework\TestCase;
+
+require __DIR__.'/../../../../delivery_monitor.civix.php';
```
Comment (akosgarai) [inline tests/phpunit/CRM/DeliveryMonitor/UpgraderTest.php:9]: Why do you need to require the file? Isn't enough to use the utils? https://docs.civicrm.org/dev/en/latest/extensions/structure/#the-big-e

```
use CRM_DeliveryMonitor_ExtensionUtil as E;
```

Comment (semseysandor) [reply tests/phpunit/CRM/DeliveryMonitor/UpgraderTest.php:9]: In the `setupHeadless()` function the extension is not installed, only in the tests. This way there is no autoloading so the class needs to be required manually.

Same as in:
https://github.com/reflexive-communications/formproc-newsletter/blob/f7f49b4dad4ba2d30be8767d353ad9b7e54f6e4b/tests/phpunit/CRM/FormprocNewsletter/UpgraderTest.php#L9

**Comment by akosgarai** [comment]
I have added a commit yesterday with changes in the documentation. Could you please give me feedback about it? Do i need to add any further info? Or should i make something more clear?

**Comment by semseysandor** [comment]
It's fine.

**Review by akosgarai — APPROVED** [review]

**Pull request merged**